hydra-node/test/Hydra/Chain/Direct/Contract/Contest.hs

```diff
@@ -50,27 +51,27 @@ healthyContestTx =
       (healthySignature healthyContestSnapshotNumber)
       (healthySlotNo, slotNoToUTCTime healthySlotNo)
       closedThreadOutput
-
-  headInput = generateWith arbitrary 42
-
-  headResolvedInput =
-    mkHeadOutput testNetworkId testPolicyId headTxOutDatum
-      & addParticipationTokens healthyParties
-
-  headTxOutDatum = toUTxOContext (mkTxOutDatum healthyClosedState)
+      (mkHeadId testPolicyId)
 
   headDatum = fromPlutusData $ toData healthyClosedState
 
-  lookupUTxO = UTxO.singleton (headInput, headResolvedInput)
+  lookupUTxO = UTxO.singleton (testSeedInput, healthyHeadTxOut)
 
   closedThreadOutput =
     ClosedThreadOutput
-      { closedThreadUTxO = (headInput, headResolvedInput, headDatum)
+      { closedThreadUTxO = (testSeedInput, healthyHeadTxOut, headDatum)
```

Comment:
Changed this to use a dedicated `TxIn` in fbd2da5ae02709fb9745cf7c901b923554878a39. It has nothing to do with the seed input after all.